src/bip85.rs

```diff
@@ -0,0 +1,63 @@
+use bdk::bitcoin::{
+    hashes::{sha512, Hash, HashEngine, Hmac, HmacEngine},
+    secp256k1::Secp256k1,
+    util::bip32::{DerivationPath, ExtendedPrivKey},
+};
+
+pub fn get_bip85_bytes<const L: usize>(xpriv: ExtendedPrivKey, path: DerivationPath) -> [u8; L] {
```

Comment:
:x: the path shouldn't be an argument here. Just an application index e.g. `330`. As it is, to get the right answer you have to pass in a path which already has `L` in it. I don't like this redundancy.